### DISCUSSION TITLE
限制节点的移动方向

### DISCUSSION THREAD

**Discussion by MrMengJ** [antvis/X6#2047] [Q&A] [answered]
是否可以限制节点的移动方向，比如只可以水平方向移动，只可以垂直方向移动

**Comment by NewByVector** [comment, marked as answer]
使用 translate: { restrict: Rectang } 配置，将节点移动范围限制在一个矩形内部。

**Reply by MrMengJ** [reply]
能否可以单独给特定的图形配置其只可以水平移动呢

**Reply by zrx1118** [reply]
是否可以禁止节点移动
